build-system/common/git.js

```diff
@@ -169,12 +169,30 @@ function gitCommitterEmail() {
 }
 
 /**
- * Returns the timestamp of the latest commit on the local branch.
+ * Returns list of commit SHAs and their cherry-pick status from master.
+ *
+ * @return {!Array<{sha: string, isCherryPick: boolean}>}
+ */
+function gitCherryMaster() {
+  return getStdout('git cherry master')
+    .trim()
+    .split('\n')
+    .map((line) => ({
+      isCherryPick: line.substring(0, 2) == '- ',
+      sha: line.substring(2),
+    }));
+}
+
+/**
+ * Returns the timestamp of a commit on the local branch.
+ *
+ * @param {string} ref a Git reference (commit SHA, branch name, etc.) for the
+ *   commit to get the time of. Leave out or set to empty string for HEAD.
  * @return {number}
  */
-function gitCommitFormattedTime() {
+function gitCommitFormattedTime(ref = '') {
```

Comment:
Would it be more explicit to set the default argument to `ref = 'HEAD'`?

Reply:
Done